packages/kolibri-components/src/content/KContentPlugin.js

```diff
@@ -0,0 +1,44 @@
+import setupPluginMediator from '../utils/pluginMediator';
+import KContentRenderer from './KContentRenderer';
+import { RENDERER_SUFFIX } from './constants';
+
+export default {
+  install(
+    Vue,
+    {
+      languageDirection,
+      ContentRendererLoadingComponent = {
+        render(h) {
+          return h('p', '-----');
+        },
+      },
+      ContentRendererErrorComponent = {
+        render(h) {
+          return h('p', 'xxxxx');
```

Comment:
what are these? could we include some comments

Reply:
Sure - these are just dummy error components.